src/components/userComponents/Footer/Footer.tsx

```diff
@@ -0,0 +1,557 @@
+            />
+          </Link>
+          <p className="text-sm font-normal">
+            The Peninsula Humane Society & SPCA (PHS & SPCA) is a local,
+            private, non-profit charitable organization dedicated to animal
+            welfare.{' '}
+          </p>
+          <Link href="https://phs-spca.org">
+            <div className="w-[200px] px-4 py-1.5 bg-hunterGreen rounded-lg">
+              <div className="text-center text-silver text-xs font-normal font-['Lato'] active:text-[#bcc0bb]">
+                Learn more about PHS/SPCA
+              </div>
+            </div>
+          </Link>
+        </div>
+        <div className="w-[150px] flex flex-col gap-4 text-ivory active:text-[#bcc0bb]">
+          <p className="text-silver s2"> VISIT </p>
+          <Link href="/hoursAdmissionPage">
+            {' '}
+            <p> Hours & Location </p>
+          </Link>
+          <Link href="/siteMapPage">
+            {' '}
+            <p> Site Maps </p>
+          </Link>
+          <Link href="/featuredToursPage">
+            {' '}
+            <p> Featured Tours</p>{' '}
+          </Link>
+        </div>
+        <div className="w-[150px] flex flex-col gap-4 text-ivory active:text-[#bcc0bb]">
+          <p className="text-silver s2"> LEARN & EXPLORE </p>
+          <Link href="/exhibitsPage">
+            {' '}
+            <p> Exhibits </p>
+          </Link>
+          <Link href="/newsFeedPage">
+            {' '}
+            <p> News </p>{' '}
+          </Link>
+          <Link href="/spotlightPage">
+            {' '}
+            <p>Wildlife Spotlights</p>{' '}
+          </Link>
+        </div>
+        <div className="flex flex-col gap-4 s1 mr-56">
+          <p className="text-silver s2"> CONTACT US </p>
+          <p className="leading-tight">
+            {' '}
+            5333 Zoo Drive <br /> Los Angeles, CA 90027{' '}
+          </p>
+          <p> (323) 644-4200 </p>
+          <p className="leading-tight">
+            {' '}
+            Open today from: <br />
+            9:30 AM – 5:00 PM{' '}
+          </p>
+          <div className="justify-start items-start gap-2.5 inline-flex">
+            <Link
+              href="https://www.facebook.com/PHSSPCA/"
+              className="w-[27px] h-[27px] justify-center items-center flex"
+            >
+              {' '}
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                x="0px"
+                y="0px"
+                width="50"
+                height="50"
+                viewBox="0 0 50 50"
+                className="fill-current text-white active:text-[#bcc0bb]"
+              >
+                <path d="M25,3C12.85,3,3,12.85,3,25c0,11.03,8.125,20.137,18.712,21.728V30.831h-5.443v-5.783h5.443v-3.848 c0-6.371,3.104-9.168,8.399-9.168c2.536,0,3.877,0.188,4.512,0.274v5.048h-3.612c-2.248,0-3.033,2.131-3.033,4.533v3.161h6.588 l-0.894,5.783h-5.694v15.944C38.716,45.318,47,36.137,47,25C47,12.85,37.15,3,25,3z" />
```

Comment:
@jxmoose is there a reason we use svgs and drawn paths instead of react icons?
Would cut down on writing + more readable this way.
https://react-icons.github.io/react-icons/search/#q=facebook